test/unittests/mongoBackend/mongoContextProvidersQuery_test.cpp

```diff
@@ -541,8 +541,9 @@ static void prepareDatabaseSeveralCprs2(void)
 *
 * noPatternAttrsAll -
 *
-* Discover:  E3 -no attrs
-* Result:    E3 - (A1, A2, A3) - http://cr1.com
+* Query:  E3 - <null>
+* Result: E3 - (A1, A2, A3) - http://cr1.com
+*
 */
 TEST(mongoContextProvidersQueryRequest, noPatternAttrsAll)
```

Comment:
Sure, I will take a close look.
However, an even closer look will 'be taken' once I start to implement my part of this whole change ...